source/snapshot/position.go

```diff
@@ -0,0 +1,52 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snapshot
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+// Position is a Snapshot Iterator's position.
+type Position struct {
```

Comment:
Maybe we have to add type of position , cdc  (c) or snapshot (s) or do you planning to have separate position structure for each iterator?

Reply:
Most likely, there will be a separate position structure for the CDC iterator.